src/colorize.cr

```diff
@@ -413,15 +416,22 @@ struct Colorize::Object(T)
         printed = true
       end
 
-      unless mode_is_default
-        # Can't reuse MODES constant because it has bold/bright duplicated
-        {% for name in %w(bold dim underline blink reverse hidden) %}
-          if mode.bits_set? MODE_{{name.upcase.id}}_FLAG
-            io << ';' if printed
-            io << MODE_{{name.upcase.id}}
-            printed = true
+      unless mode.none?
+        printed_bright = false
+        mode.each do |flag|
+          # Enum#each yields each member flag. Bright and bold have the same value
+          # and would show up as duplicate, so we need to handle this special case.
+          if flag.bright?
+            if printed_bright
+              next
+            else
+              printed_bright = true
+            end
           end
```

Comment:
Consider this refactor: https://github.com/oprypin/crystal/commit/5956803b29c408a64f9ac3c00810151b57e2d047